rapid7_insightvm/help.md

```diff
@@ -2001,269 +2001,6 @@ Example Output:
 }
 ```
 
-Example output:
```

Comment:
@mhofert-r7 did this example output get removed accidentally?

Reply:
there were 2 example outputs